frontend/app/services/IdentityService.scala

```diff
@@ -24,18 +26,17 @@ import views.support.IdentityUser
 
 import scala.concurrent.Future
 import scala.concurrent.duration._
-import scala.util.{Failure, Success}
-import dispatch._
-import Defaults.timer
-import play.api.http.Status
-import play.api.mvc.Results
 
 case class IdentityServiceError(s: String) extends Throwable {
   override def getMessage: String = s
 }
 
-object IdentityService {
-  val DisregardResponseContent: (WSResponse => Either[String, Unit]) = resp => Right(Unit)
+object IdentityService extends LazyLogging {
+  val DisregardResponseContent: (WSResponse => Either[String, Unit]) = {
+    resp =>
+      logger.debug(s"Webservice returned code ${resp.status}, and body: ${resp.body}")
```

Comment:
Is this debug statement still needed?

Reply:
I think it can be useful because otherwise this method completely throws away the response, even if it is an error.